### ISSUE TITLE
Morph requires experimental nix-commands to be enabled

### ISSUE THREAD

**Issue by mweinelt** [DBCDK/morph#173]
Very recently nix in nixos-unstable has been upgrade to 2.4. Now I am forced to enable the nix-command, which is deemed experimental.

```
❯ morph deploy default.nix switch
error: experimental Nix feature 'nix-command' is disabled; use '--extra-experimental-features nix-command' to override
```

**Comment by srhb** [comment]
Related to https://github.com/DBCDK/morph/issues/156 -- I think we might give making morph compatible with either version a stab, but I'm not sure how that will work out quite yet. I remember that fixing logs with nix-command was easier than fixing logs without. Maybe this has changed in 2.4, which would be a nicer requirement than experimental features.

**Comment by srhb** [comment]
This should be fixed in #175 (pending release)

Let us know if it's not the case. :slightly_smiling_face:

**Comment by NoraCodes** [comment]
I get the same error message on Nix 2.8 with Morph 1.6.0 from `nixos-21.11`.

**Comment by Flinner** [comment]
can confirm with NoraCodes, gettings this error

```
error: experimental Nix feature 'nix-command' is disabled; use '--extra-experimental-features nix-command' to override
Error while running `nix eval ..`: exit status 1
```

**Comment by srhb** [comment]
The fix is included in morph 1.7.0, which is not in 21.11, but is in unstable (and thus the upcoming 22.05). Sorry for the inconvenience. :)
